assets/alice.gui

```diff
@@ -122,47 +118,78 @@ guiTypes = {
 			}
 			guiButtonType = {
 				name = "window_mode_checkbox"
-				position = { 51 115 }
```

Comment:
this was intentional so that the text would be vertically centered with respect to the button